src/oracle/oracle.cairo

```diff
@@ -1492,7 +1463,8 @@ mod Oracle {
                                 )
                             );
                     },
-                    DataType::GenericEntry(_) => {},
+                    DataType::GenericEntry(key) => { //NEED TO BE DONE BASED ON USE CASE
```

Comment:
please create an issue and add it to the comment `TODO: #<issue_number>`

Reply:
Ok